src/libasr/pass/print_arr.cpp

```diff
@@ -85,7 +85,7 @@ class PrintArrVisitor : public PassUtils::PassVisitor<PrintArrVisitor>
                 doloop_body.push_back(al, doloop);
                 doloop_body.push_back(al, empty_print_endl);
             }
-            doloop = ASRUtils::STMT(ASR::make_DoLoop_t(al, loc, head, doloop_body.p, doloop_body.size()));
+            doloop = ASRUtils::STMT(ASR::make_DoLoop_t(al, loc, nullptr, head, doloop_body.p, doloop_body.size()));
```

Comment:
Changes in `pass/print_arr.cpp` are missing, [lfortran/lfortran@b1992c3](https://github.com/lfortran/lfortran/pull/1370/commits/b1992c3bec1c2ee65e7f510ba4b9ff960779113f)

Reply:
Yup, I shared here https://github.com/lcompilers/lpython/pull/1670#issuecomment-1498628934.

Reply:
Again, that's a very old commit. I just copied today mornings `main` from `lfortran` into `lpython`.

Reply:
It is there in the latest main as well.

Reply:
https://github.com/lfortran/lfortran/blob/main/src/libasr/pass/print_arr.cpp#L138

Reply:
Yes. I added it in this PR.